javatests/arcs/android/systemhealth/testapp/StorageCore.kt

```diff
@@ -199,7 +199,14 @@ class StorageCore(val context: Context, val lifecycle: Lifecycle) {
                         Thread.sleep(GcWaitTimeMs)
 
                         taskManagerEvents.queue.add(
-                            TaskEvent(TaskEventId.MEMORY_STATS, 0, MemoryStats.appJvmHeapKbytes)
+                            TaskEvent(
+                                TaskEventId.MEMORY_STATS,
+                                0,
+                                listOf(
+                                    MemoryStats.appJvmHeapKbytes,
+                                    MemoryStats.appNativeHeapKbytes,
+                                    MemoryStats.allHeapsKbytes
+                                ))
```

Comment:
I think the closing parens of TaskEvent is supposed to be on a line by itself in the same column as the 'T' of TaskEvent

Reply:
Done.